openhands/core/main.py

```diff
@@ -122,7 +122,7 @@ async def run_controller(
 
     if runtime is None:
         runtime = create_runtime(config, sid=sid)
-    await runtime.connect()
+        await runtime.connect()
```

Comment:
This is necessary, otherwise the pass in runtime from evaluation will get connected twice